### PR TITLE
Update with-visual-studio-code.md

### PR DESCRIPTION
**Pull request by jambanks91**
## Summary

Opening my file

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/core/tutorials/with-visual-studio-code.md [hunk 1]
```diff
@@ -86,15 +86,15 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
...
+2. Name your file *Fluffy-broccolli.cs*. You must save it with a `.cs` extension at the end for it to be recognized as a csharp file.
 3. Add the code below to create your first class. Make sure to include the correct namespace so you can reference it from your *Program.cs* file:
 
     ``` csharp
     using System;
 
     namespace HelloWorld
     {
-        public class MyClass
+        public class Fluffy-broccolli
```
Comment (Youssef1313) [inline docs/core/tutorials/with-visual-studio-code.md]: I don't think that dashes are allowed in identifier names such as classes.

File: docs/core/tutorials/with-visual-studio-code.md [hunk 2]
```diff
@@ -115,8 +115,9 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
         {
             static void Main(string[] args)
             {
-                var c1 = new MyClass();
-                Console.WriteLine($"Hello World! {c1.ReturnMessage()}");
+                var c1 = new Fluffy-broccolli();
+                            Console.WriteLine("Export-1910190154231820315064419971 (1).zip https://pxsprod01.blob.core.windows.net/6cf49368d7ec1b41d23a43feffffffff/Export-1910190154231820315064419971.zip?sv=2018-03-28&sr=b&sig=rnwuBK6EWeMnV7V0SzzzWiiDzOHrghvodkc92CjWyqA%3D&spr=https&se=2019-10-26T01%3A54%3A53Z&sp=r;
```
Comment (Youssef1313) [inline docs/core/tutorials/with-visual-studio-code.md]: Indentation is bad here. And Also I think the line is mistakenly splitted into two lines.

**Comment by mairaw** [comment]
I don't understand the goal of these proposed changes. Do you mind explaining them @jambanks91?

**Comment by Youssef1313** [comment]
> 
> 
> @jambanks91 The link you have posted in this code looks like personal data related to cortana usage (or something) we're unsure why you've opened this PR but I suggest that you delete your github fork to erase that data. I've taken the liberty to at least change the link so that any web indexing that gets done doesn't have the current valid zip.
> 
> You can delete your fork by visiting https://github.com/jambanks91/docs/settings and clicking delete at the bottom of the page.

I've also deleted a review of mine that was still have the valid link.

**Pull request closed without merging**